client/src/modules/templates/bhCurrencySelect.tmpl.html

```diff
@@ -1,5 +1,11 @@
-<ng-form name="$ctrl.form">
-  <div class="radio" ng-class="{ 'has-error' : ($ctrl.validationTrigger && $ctrl.form.$invalid) || !$ctrl.valid }" data-bh-currency-select>
+<fieldset
+  ng-form="CurrencySelectForm"
+  novalidate
+  bh-currency-select>
```

Comment:
Looks like this needs to have "data-" appended to it for the cash tests to pass!

Reply:
Ok, pushed a fix to this.  Lets see if it works!